src/selectors/breakpointAtLocation.js

```diff
@@ -66,3 +76,10 @@ export function getBreakpointAtLocation(state, location) {
 
   return findBreakpointAtLocation(breakpoints, selectedSource, location);
 }
+
+export function getBreakpointsAtLine(state: OuterState, line: number) {
+  const selectedSource = getSelectedSource(state);
+  const breakpoints = getBreakpointsForSource(state, selectedSource);
+
+  return findBreakpointsAtLine(breakpoints, selectedSource, line);
```

Comment:
minor nit, it might be nice to inline this function as it's only used in one place:


```js
  return breakpoints.filter(
    breakpoint => getLocation(breakpoint, selectedSource).line === line
  );
```

Reply:
Sure.